sys-devel/m4_bootstrap/m4_bootstrap-1.4.19.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A macro processor"
+DESCRIPTION="
+	GNU M4 is an implementation of the traditional Unix macro processor.
+	It is mostly SVR4 compatible although it has some extensions (for
+	example, handling more than 9 positional parameters to macros).
+	GNU M4 also has built-in functions for including files, running shell
+	commands, doing arithmetic, etc.
+
+	M4 is a macro processor in the sense that it copies its input to the
+	output expanding macros as it goes. Macros are either builtin or
+	user-defined and can take any number of arguments. Besides just doing
+	macro expansion, m4 has builtin functions for including named files,
+	running UNIX commands, doing integer arithmetic, manipulating text in
+	various ways, recursion etc... M4 can be used either as a front-end to
+	a compiler or as a macro processor in its own right.
+
+	One of the biggest users of M4 is the GNU Autoconf project.
+	"
+HOMEPAGE="http://www.gnu.org/software/m4/"
+SOURCE_URI="http://ftp.gnu.org/gnu/m4/m4-1.4.19.tar.gz"
+LICENSE="GNU GPL v3"
+COPYRIGHT="2000, 2005-2011 Free Software Foundation, Inc."
+CHECKSUM_SHA256="3be4a26d825ffdfda52a56fc43246456989a3630093cced3fbddf4771ee58a70"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64 ppc arm arm64 riscv64 sparc m68k"
```

Comment:
maybe enable it only for PowerPC until the other architectures can be tested with it?

Reply:
We are on 1.4.19 in HaikuPorts for all architectures, let's just go with this for all architectures here.

I think we should use `all` here though.

Reply:
Confirmed needed for arm64 as well.

We currently don't use "all" much on haikuports.cross, to try to keep it clear what has been tested on each architecture. So I'm going to merge this as-is for now.